partner_manual_rank/readme/CREDITS.rst

```diff
@@ -0,0 +1,2 @@
+ForgeFlow, S.L.
+Vauxoo
```

Comment:
To the RST syntax, a new line is equivalent to a whitespace. If you want to add several items, you should use a list.

You may check how the resulting README was generated